### DISCUSSION TITLE
External Library as Album

### DISCUSSION THREAD

**Discussion by swap83** [immich-app/immich#4186] [General]
It would be nice if every time I insert a New External Library, it becomes an album.

**Comment by PubBow** [comment]
Currently using PhotoView which does seem to work well with adding existing organised folders with in return creates the Albums to match.  I also allows different users to have access to the same and different albums which is also great.

However, it seems development has stopped and it doesn't always show the correct number of photos and videos that exist which is odd and it also is quite behind with the iOS app.

I have found Immich which looks great, a much more polished app however not being able to add existing existing organised folders and auto creating the albums to match is stopping me from using.

If this was an option (unless I'm missing it) I would defo move over to Immich.

**Comment by andymarden** [comment]
Definitely upvote this. I have a nas with hundreds of folder on it which are essentially "albums" in my barebones world. It make a lot if sense with an external library to be able to create albums based on the folder structure.

**Comment by traktuner** [comment]
This would be so cool!

**Comment by aviv926** [comment]
There seems to be something almost similar already in progress
#4279
